core/templates/pages/about-page/about-page.component.spec.ts

```diff
@@ -16,161 +16,208 @@
  * @fileoverview Unit tests for the about page.
  */
 
+import { ComponentFixture, fakeAsync, TestBed} from '@angular/core/testing';
+import { EventEmitter, NO_ERRORS_SCHEMA, Pipe, PipeTransform }
+  from '@angular/core';
+
+import { AboutPageComponent } from './about-page.component';
+import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
+import { TranslateService } from 'services/translate.service';
+import { UtilsService } from 'services/utils.service';
+import { UrlInterpolationService } from
```

Comment:
I am a bit confused here, sorted by:

- what is imported or
- from where it is imported?

So will Urlinterpolation import go up because it is imported from `domain/utilities/url-interpolation.service` and `domain` < `services`?